pkg/controller/disruption/config/v1alpha1/defaults.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	kubectrlmgrconfigv1alpha1 "k8s.io/kube-controller-manager/config/v1alpha1"
+)
+
+// RecommendedDefaultDisruptionControllerConfiguration sets the defaults for DisruptionController.
+func RecommendedDefaultDisruptionControllerConfiguration(obj *kubectrlmgrconfigv1alpha1.DisruptionControllerConfiguration) {
+	if obj.ConcurrentDisruptionSyncs == 0 {
+		obj.ConcurrentDisruptionSyncs = 5
+	}
+	if obj.ConcurrentDisruptionStalePodSyncs == 0 {
+		obj.ConcurrentDisruptionStalePodSyncs = 5
+	}
+}
```

Comment:
Why did we choose 5 as a default here? Is it a good idea to change the defaults and spin up 10 goroutines for current users?

Reply:
The current concurrency value refers to the default value of 5 used by the Deployment controller.
https://github.com/kubernetes/kubernetes/blob/71f0fc6e72d53d5caf50b1314ca4d754463117f0/pkg/controller/deployment/config/v1alpha1/defaults.go#L34

Reply:
the default value should be the same as it used today, that IIUIC is 1

Reply:
@aojea done. Agreed. The ConcurrentDisruptionSyncs and ConcurrentDisruptionStalePodSyncs parameters have been set to their original default value of 1.